tests/src/test/scala/whisk/core/controller/test/WebActionsApiTests.scala

```diff
@@ -328,6 +338,7 @@ trait WebActionsApiTests extends ControllerTestCommon with BeforeAndAfterEach wi
 
         it should s"not match invalid routes (auth? ${creds.isDefined})" in {
             implicit val tid = transid()
+            customOptions = true
```

Comment:
The majority of the tests invoke a web action with the OPTIONS verb. In order to maintain the same test behavior I am adding the custom options annotation to those tests. We could enable the annotation by default in the tests when declaring the custonOptions value.